charts/flame-hub/scripts/monitor-pg-migration.sh

```
@@ -0,0 +1,118 @@
+
+  printf '%s\n' '--- non-zero tables only ---'
+  for db in "${EXPECTED_DBS[@]}"; do
+    run_psql "$POD" -d "$db" -Atc \
+      "SELECT '${db}.'||schemaname||'.'||relname||E'\t'||n_live_tup FROM pg_stat_user_tables WHERE n_live_tup > 0 ORDER BY n_live_tup DESC;" \
+      2>/dev/null || true
+  done
+} | tee "$SNAPSHOT"
+
+# One-line summary for diffing across phases
+db_count="$(grep -E '^[^-]' "$SNAPSHOT" | awk -F'\t' '/^auth\t|^core\t|^registry\t|^storage\t|^telemetry\t/{c++} END{print c+0}')"
+nonzero="$(grep -c $'\t' "$SNAPSHOT" || true)"
```

Shellcheck (0.11.0)

[warning] nonzero appears unused. Verify use (or export if used externally).

(SC2034)